FreeRTOS/Test/CMock/smp/single_priority_no_timeslice/covg_single_priority_no_timeslice_utest.c

```diff
@@ -415,109 +415,3 @@ void test_task_step_tick_xNextTaskUnblockTime_not_equal( void )
     xTickCount = 0U;
     vTaskStepTick((TickType_t)10U);
 }
-
```

Comment:
Yes, those may be duplicate. In fact they were meant to cover this:

          /* Ready lists can be accessed so move the task from the
                         * suspended list to the ready list directly. */
                        if( pxTCB->uxPriority >= pxCurrentTCB->uxPriority )
                        {
                            xYieldRequired = pdTRUE;

                            /* Mark that a yield is pending in case the user is not
                             * using the return value to initiate a context switch
                             * from the ISR using portYIELD_FROM_ISR. */
                            xYieldPendings[ 0 ] = pdTRUE;
                        }
                        else
                        {
                            mtCOVERAGE_TEST_MARKER();
                        }

However, the above is within an IFDEF and only applies to single core processing. So I think they are not actually covering the case I intended, and that case is already covered by other tests.

Reply:
![image](https://user-images.githubusercontent.com/61685396/220244774-5053c0c2-0de5-4dcd-bbf4-1f10db9690bf.png)
These test cases tried to cover these section in the image. The comment also have code snippet to indicate that.
I run the coverage test with you repo. They are not covered.
So I don't think these are duplicated test cases.
Suggest we don't remove these test cases.